test/parallel/test-fs-chmod-mask.js

```diff
@@ -89,6 +90,32 @@ function test(mode, asString) {
     fs.lchmodSync(link, input);
     assert.strictEqual(fs.lstatSync(link).mode & 0o777, mode);
   }
+
```

Comment:
Could you add a comment about what this tests:
```js
  // Test that `lchmod` works on readonly (0x444) files.
```

Reply:
added, thanks 👍